tests/jq.test

```diff
@@ -1547,12 +1547,22 @@ ascii_upcase
 "useful but not for é"
 "USEFUL BUT NOT FOR é"
 
-bsearch(0,2,4)
+bsearch(0,1,2,3,4)
 [1,2,3]
 -1
+0
 1
+2
 -4
 
+bsearch({x:1})
+[{ "x": 0 },{ "x": 1 },{ "x": 2 }]
+1
+
+bsearch(0)
+"aa"
+jq: error (at <unknown>): string ("aa") cannot be searched from
```

Comment:
```suggestion
try ["OK", bsearch(0)] catch ["KO",.]
"aa"
["KO","string (\"aa\") cannot be searched from"]
```
Also you used a number as target which is not necessarily allocated, you should use something like an array or object, with elements